freqtrade/resolvers/iresolver.py

```diff
@@ -29,7 +29,8 @@ def _get_valid_object(object_type, module_path: Path,
         """
 
         # Generate spec based on absolute path
-        spec = importlib.util.spec_from_file_location('unknown', str(module_path))
+        # Pass object_name as first argument to have logging print a reasonable name.
+        spec = importlib.util.spec_from_file_location(object_name, str(module_path))
```

Comment:
This change should lay disguising somewhere in my old stashes that are in the queue for handling... 😄